openedx/core/djangoapps/user_api/accounts/api.py

```diff
@@ -67,7 +69,7 @@ def get_account_settings(request, username=None, configuration=None, view=None):
         username = requesting_user.username
 
     try:
-        existing_user = User.objects.get(username=username)
+        existing_user = User.objects.select_related('preference').get(username=username)
```

Comment:
As per the discussion on *HipChat/Mobile Server*, this doesn't do anything, because preference is on the many side of a FK relationship. However, running `select_related('profile')` does save a query.  

Also, it looks like there are more queries being run by the tests than are run when accessing the actual API view.  The actual API view makes 3 queries with `select_related('profile')` and 4 without.  See the output from django-debug-toolbar at:

<img width="1286" alt="screen shot 2015-10-06 at 4 34 45 pm" src="https://cloud.githubusercontent.com/assets/77486/10321917/4fbd757c-6c49-11e5-8a29-97684316a624.png">

We may want to investigate where the extra queries are coming from in the test, but I think the view itself looks reasonable.